### DISCUSSION TITLE
upload photos in discord

### DISCUSSION THREAD

**Discussion by hamidmirzaei282** [community/community#146165] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can't upload the picture in discord, does anyone know where the problem is? Tell me, what settings should I make in v2ray so that I can upload photos to Discord?

**Comment by mecodeatlas** [comment]
Hi @hamidmirzaei282,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Discord Support](https://support.discord.com/hc/en-us). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
